src/stores/CallStore.ts

```diff
@@ -187,7 +184,4 @@ export class CallStore extends AsyncStoreWithClient<EmptyObject> {
     };
 
     private onGroupCall = (groupCall: GroupCall): void => this.updateRoom(groupCall.room);
-    private onRTCSessionStart = (roomId: string, session: MatrixRTCSession): void => {
-        this.updateRoom(session.room);
-    };
```

Comment:
Can we really remove this? Why was the updateRoom required and now is not anymore?

Reply:
if this is tested and does indeed work because of the widget update than this is nice to get cleaned up.